ahoopes16/zombie-roleplaying-server
# Create Weapons

### Summary
This PR implements a route for creating weapon objects using the API.

If accepted, this PR will:

- Add a route for creating weapon objects and saving them to the database

### To Test

1. Run `yarn start`
1. Import `tsoa/swagger.json` into Postman
1. Hit http://localhost:8000/weapons with the provided body
1. You should get the result back in the API
